### DISCUSSION TITLE
push to github

### DISCUSSION THREAD

**Discussion by sriramab** [koordinates/kart#919] [Q&A] [answered]
Hi, 
Is it possible to push to github ? I do not clearly understand how to set the remote for my local kart repo. Please guide.

**Comment by tcwilkinson** [comment, marked as answer]
Kart seems to follow git syntax here (or pass on directly to git), so you're looking for something like following:-

```
kart remote add origin https://github.com/user/repository
kart push
```

The repo will obviously look a little different on github though from your working copy!
